minecraft_servers/griefergames/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "GrieferGames",
```

Comment:
As it is already requested, the `server_name` has to be the directory name.

Reply:
Yeah, i changed the server name